### PR TITLE
Update A1/A3 info

### PR DESCRIPTION
**Pull request by outofxxx**
1. Small refactoring for A1.
2. According to the update of this [document](https://docs.google.com/spreadsheets/d/1k2ZvrLK5v_KO_tsCWJVtjbs8TS8PzM_idWv64z4ynSI/edit?pli=1#gid=1065809408), keep A1/A3 consistent with its content.

### CODE REVIEW

File: tee-worker/litentry/core/assertion-build/src/a3.rs [hunk 1]
```diff
@@ -24,8 +24,8 @@ use crate::*;
 use lc_data_providers::{discord_litentry::DiscordLitentryClient, vec_to_string};
 
 const VC_A3_SUBJECT_DESCRIPTION: &str =
-	"The user has commented in a specific Discord channel with a specific role";
-const VC_A3_SUBJECT_TYPE: &str = "Discord Member Verification";
+	"You have commented in Litentry Discord #🪂id-hubber channel. Channel link: https://discord.com/channels/807161594245152800/1093886939746291882";
```
Comment (Kailai-Wang) [inline tee-worker/litentry/core/assertion-build/src/a3.rs:27]: Is this emoji intended? 😲

File: tee-worker/litentry/core/assertion-build/src/a1.rs [hunk 2]
```diff
@@ -23,31 +23,23 @@ extern crate sgx_tstd as std;
...
 
 	match Credential::new(&req.who, &req.shard) {
 		Ok(mut credential_unsigned) => {
-			// add subject info
 			credential_unsigned.add_subject_info(VC_A1_SUBJECT_DESCRIPTION, VC_A1_SUBJECT_TYPE);
-
-			// add assertion
-			let flag = web2_cnt != 0 && web3_cnt != 0;
-			credential_unsigned.add_assertion_a1(flag);
+			credential_unsigned.add_assertion_a1(is_web2 && is_web3);
```
Comment (Kailai-Wang) [inline tee-worker/litentry/core/assertion-build/src/a1.rs:42]: The logic doesn't change, right? So we still require the user links at least 1 web2 AND 1 web3?

File: tee-worker/litentry/core/assertion-build/src/a3.rs (hunk 1 above)
Comment (outofxxx) [reply tee-worker/litentry/core/assertion-build/src/a3.rs:27]: Yes, after confirming with Kayla again, this was intentionally added.

File: tee-worker/litentry/core/assertion-build/src/a1.rs (hunk 2 above)
Comment (outofxxx) [reply tee-worker/litentry/core/assertion-build/src/a1.rs:42]: Yes, The logic of A1 has not changed, but I refactored the code here.

**Review by Kailai-Wang — APPROVED** [review]

**Review by BillyWooo — APPROVED** [review]
LGTM.

**Pull request merged**